PHPStan is not required and phpstan.installer.neon is missing

- [x] I was not able to find an [open](https://github.com/zendframework/zend-expressive-skeleton/issues?q=is%3Aopen) or [closed](https://github.com/zendframework/zend-expressive-skeleton/issues?q=is%3Aclosed) issue matching what I'm seeing.
 - [x] This is not a question. (Questions should be asked on [chat](https://zendframework.slack.com/) ([Signup for here](https://zendframework-slack.herokuapp.com/)) or our [forums](https://discourse.zendframework.com/).)

The documentation states:
>PHPUnit and PHP_CodeSniffer are now installed by default. To execute tests and detect coding standards violations, run the following command:
>
> `$ composer check`

However the command runs into an error (see below). To get it up and running one needs to composer require (dev) phpstan and create an empty `phpstan.installer.neon` file.

Since I'm new to Zend Expressive, I'm not sure, if the fix needs to be done in the code or the documentation is off. Please let me know, if this belongs to the docs. Thanks in advance.

### Code to reproduce the issue

<!-- Please provide the minimum code necessary to recreate the issue -->

```bash
$ composer create-project zendframework/zend-expressive-skeleton expressive
$ cd expressive
$ composer check
```

### Expected results

A valid response of the phpstan check, something like

```bash
# [Test results of PHPUnit]

> phpstan analyze -l max -c ./phpstan.installer.neon ./src ./config


 ------ ------------------------------------------------------------
  src\App\src\Handler\HomePageHandler.php                     12/12 [============================] 100%
 ------ ------------------------------------------------------------
  70     Class Zend\Expressive\Router\AuraRouter not found.
  76     Class Zend\Expressive\Router\ZendRouter not found.
  81     Class Zend\Expressive\Plates\PlatesRenderer not found.
  84     Class Zend\Expressive\Twig\TwigRenderer not found.
  87     Class Zend\Expressive\ZendView\ZendViewRenderer not found.
 ------ ------------------------------------------------------------

 ------ --------------------------------------------------------
  Line   config\config.php
 ------ --------------------------------------------------------
  26     Class Zend\Expressive\Swoole\ConfigProvider not found.
  27     Class Zend\Expressive\Swoole\ConfigProvider not found.
 ------ --------------------------------------------------------

Script phpstan analyze -l max -c ./phpstan.installer.neon ./src ./config handling the analyze event returned with error code 1


 [ERROR] Found 7 errors


Script @analyze was called via check
```

### Actual results

```bash
# [Test results of PHPUnit]

> phpstan analyze -l max -c ./phpstan.installer.neon ./src ./config
Project config file at path ./phpstan.installer.neon does not exist.
Script phpstan analyze -l max -c ./phpstan.installer.neon ./src ./config handling the analyze event returned with error code 1
Script @analyze was called via check
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -22,7 +22,7 @@ All notable changes to this project will be documented in this file, in reverse 
 
 ### Fixed
 
-- Nothing.
+- [#264](https://github.com/zendframework/zend-expressive-skeleton/pull/264) removes phpstan leftovers.
 
 ## 3.2.0 - 2018-09-27
 

--- a/src/ExpressiveInstaller/OptionalPackages.php
+++ b/src/ExpressiveInstaller/OptionalPackages.php
@@ -400,7 +400,8 @@ class OptionalPackages
 
         // Remove phpstan completely
         $this->composerDefinition['scripts']['check'] = array_diff(
-            $this->composerDefinition['scripts']['check'], ['@analyze']
+            $this->composerDefinition['scripts']['check'],
+            ['@analyze']
         );
         unset($this->composerDefinition['scripts']['analyze']);
     }

--- a/src/ExpressiveInstaller/OptionalPackages.php
+++ b/src/ExpressiveInstaller/OptionalPackages.php
@@ -397,6 +397,12 @@ class OptionalPackages
         // Remove installer scripts
         unset($this->composerDefinition['scripts']['pre-update-cmd']);
         unset($this->composerDefinition['scripts']['pre-install-cmd']);
+
+        // Remove phpstan completely
+        $this->composerDefinition['scripts']['check'] = array_diff(
+            $this->composerDefinition['scripts']['check'], ['@analyze']
+        );
+        unset($this->composerDefinition['scripts']['analyze']);
     }
 
     /**

--- a/test/ExpressiveInstallerTest/RemoveInstallerTest.php
+++ b/test/ExpressiveInstallerTest/RemoveInstallerTest.php
@@ -49,5 +49,7 @@ class RemoveInstallerTest extends OptionalPackagesTestCase
         $this->assertFalse(isset($composer['extra']['optional-packages']));
         $this->assertFalse(isset($composer['scripts']['pre-install-cmd']));
         $this->assertFalse(isset($composer['scripts']['pre-update-cmd']));
+        $this->assertFalse(isset($composer['scripts']['check']['@analyze']));
+        $this->assertFalse(isset($composer['scripts']['analyze']));
     }
 }

